include/libtorrent/alert_types.hpp

```diff
@@ -2496,7 +2537,7 @@ namespace libtorrent
 #undef TORRENT_DEFINE_ALERT_PRIO
 #undef TORRENT_CLONE
 
-	enum { num_alert_types = 89 };
+	enum { num_alert_types = 90 };
```

Comment:
The other changes are beacouse `num_alert_types` is not dynamic. It seems to me a little odd not having the right amount of alert types even if `TORRENT_DISABLE_LOGGING` is defined. Specifically for picker_log_alert there was no constructor if `TORRENT_DISABLE_LOGGING` and `peer_alert` has no default constructor.